src/ChargeDriftModels/ADL/ADL.jl

```diff
@@ -116,18 +151,40 @@ function ADLChargeDriftModel(configfilename::Union{Missing, AbstractString} = mi
     h111 = VelocityParameters{T}(h111mu0, h111beta, h111E0, 0)
     electrons = CarrierParameters{T}(e100, e111)
     holes     = CarrierParameters{T}(h100, h111)
+    
+    
+    if "material" in keys(config)
+        config_material::String = config["material"]
+        if config_material == "HPGe"
+            material = HPGe
+        elseif config_material == "Si"
+            material = Si
+        else
+            @warn "Material \"$(config_material)\" not supported for ADLChargeDriftModel.\nSupported materials are \"Si\" and \"HPGe\".\nUsing \"$(Symbol(material))\" as default."
+        end
+    end
 
-    if "masses" in keys(config["drift"])
+    parameters = if "masses" in keys(config["drift"])
         ml = T(config["drift"]["masses"]["ml"])
         mt = T(config["drift"]["masses"]["mt"])
-        masses = MassParameters{T}(ml, mt)
+        ADLParameters{T}(ml, mt, material)
     else
-        masses = MassParameters{T}(T(1.64),T(0.0819))
+        ml = T(material_properties[Symbol(material)].ml)
+        mt = T(material_properties[Symbol(material)].mt)
+        ADLParameters{T}(ml, mt, material)
     end
 
-    ismissing(phi110) ? phi110 = T(config["phi110"]) : phi110 = T(phi110)  #if you give the angle of the 110 axis it will be used, otherwise read from config file
+    crystal_orientation::SMatrix{3,3,T,9} = if ismissing(phi110) 
+        if "phi110" in keys(config)
+            RotZ{T}(- π/4 - config["phi110"] )
+        else
+            transpose(parse_rotation_matrix(T, config, u"rad")) # replace u"rad" with the units in the config file
+        end
+    else
+        RotZ{T}(- π/4 - phi110)
+    end
```

Comment:
This is where `crystal_orientation`  gets parsed.
`"X": x` in the config file will rotate the crystal axes by `x` radians around the `X` axis.

Right now, the value `x` is **always** parsed in units of radians (see L. 181).
This should be changed when updating the config file format (#136) and making the `charge_drift_model` a part of a `Semiconductor`. Then, the user-defined units in the config file should be chosen.